types/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import _Vue, { PluginFunction } from "vue";
```

Comment:
Why prefix with `_`?

Reply:
Ah sorry, initially I had typed the `install` method (like [this](https://github.com/vuejs/vuex/blob/7807af7fcb99d4fc30be1c9c721a250614d77a88/types/index.d.ts#L42)), but I've removed that so I'll remove the prefix.